src/js/components/picklist/index.js

```diff
@@ -53,11 +53,16 @@ const PicklistEmpty = View.extend({
   className: 'picklist__message',
   template: hbs`
     {{#if isLoading}}
-      {{ loadingText }}
+      <div class="picklist__message-loading js-loading">
+        {{fas "circle"}}{{ loadingText }}
+      </div>
     {{else}}
       {{ noResultsText }}
     {{/if}}
   `,
+  ui: {
```

Comment:
looks like we can remove the selector and ui element?

Reply:
Yep, not needed anymore

Reply:
That's been removed now 👍